dockers/docker-auditd/config_checker.py

```diff
@@ -46,18 +47,23 @@ def run_command(cmd):
     return p.returncode, error
 
 
-def get_hwsku():
-    hwsku_cmd = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
-    rc, out = run_command(hwsku_cmd)
+def get_arch():
+    arch_cmd = NSENTER_CMD + "uname -m"
+    rc, out = run_command(arch_cmd)
```

Comment:
>arch_cmd [](http://example.com/codeflow?start=26&length=8)

Using array of string will be safer. In this case since the final command is fixed, not related to user input, this suggest is optional.